modules/packages/TOML.chpl

```diff
@@ -354,7 +354,8 @@ module TomlParser {
         }
         // DateTime
         else if dt.match(val) {
-          var date = datetime.strptime(getToken(source), "%Y-%m-%dT%H:%M:%SZ");
+          var date : datetime;
+          date.strptime(getToken(source), "%Y-%m-%dT%H:%M:%S");
```

Comment:
I guess we would be required to handle error (?) as well, as `strptime` now throws

Reply:
> I guess we would be required to handle error (?) as well, as `strptime` now throws

Not in that particular test case, I don't think. If we are changing `strptime` to throw instead of returning an error, then we might need to change the test case to expect an error to be thrown, but I did not see it expecting an error. It looked like the extra character in the format was just expected to be ignored.

Reply:
ohh got it